.github/workflows/release.yaml

```diff
@@ -1,8 +1,11 @@
 name: Release
 on:
-  push:
-    tags:
-      - v[0-9]+.[0-9]+.[0-9]+
+  # Only run when this repo changes from private to public.
+  # Already public = never
+  public
```

Comment:
The is the least disruptive way to turn off workflows.  We still want to be able to easily merge changes, so we don't want to delete the files.